terraform/etcd.tf

```diff
@@ -0,0 +1,10 @@
+module "etcd" {
```

Comment:
What do you think about using existing module from CoreOS https://github.com/coreos/tectonic-installer/tree/track-1/modules/aws/etcd?

Reply:
This one will be harder to use as it is. Etcd is integrated with the k8s installer and has some internal dependencies, like dns, certs or s3 bucket with init scripts.
I need to take a deeper look.